src/Rokt-Kit.js

```diff
@@ -136,6 +140,31 @@ var constructor = function () {
     this.selectPlacements = selectPlacements;
 
     // mParticle Kit Callback Methods
+    function fetchOptimizely() {
+        const forwarders = window.mParticle._getActiveForwarders().filter(function (forwarder) {
+            return forwarder.name === 'Optimizely';
+        });
+
+        try {
+            if (forwarders.length > 0) {
+                // Get the state object
+                var optimizelyState = window.optimizely.get('state');
+                // Get active experiment IDs
+                var activeExperimentIds = optimizelyState.getActiveExperimentIds();
+                // Get variations for each active experiment
+                var activeExperiments = activeExperimentIds.reduce(function(acc, expId) {
+                    acc[`rokt.custom.optimizely.${expId}.variationId`] = optimizelyState.getVariationMap()[expId].id;
+                    return acc;
+                }, {});
+
+                console.log("[ROKT] Active Optimizely Experiments:", activeExperiments);
```

Comment:
Nit: Is this console log just for testing or would we want this to show up in production for Partners to review? Not sure if it's necessary for us to have a "debug" or "verbose" logging setting to hide these.